doc/sphinx/magnetostatics.rst

```diff
@@ -111,30 +111,30 @@ Due to the long-range nature of dipolar interactions, Direct summation with mini
 
 Two methods are available:
 
-* :class:`espressomd.magnetostatics.DipolarDirectSumCpu`
+* :class:`~espressomd.magnetostatics.DipolarDirectSumCpu`
   performs the calculation in double precision on the Cpu.
 
 
-* :class:`espressomd.magnetostatics.DipolarDirectSumGpu`
+* :class:`~espressomd.magnetostatics.DipolarDirectSumGpu`
   performs the calculations in single precision on a Cuda-capable graphics card.
   The implementation is optimized for large systems of several thousand
   particles. It makes use of one thread per particle. When there are fewer
   particles than the number of threads the gpu can execute simultaneously,
   the rest of the gpu remains idle. Hence, the method will perform poorly
   for small systems.
 
-To use the methods, create an instance of either :class:`espressomd.magnetostatics.DipolarDirectSumCpu` or :class:`espressomd.magnetostatics.DipolarDirectSumGpu` and add it to the system's list of active actors. The only required parameter is the Prefactor :eq:`dipolar_prefactor`::
+To use the methods, create an instance of either :class:`~espressomd.magnetostatics.DipolarDirectSumCpu` or :class:`~espressomd.magnetostatics.DipolarDirectSumGpu` and add it to the system's list of active actors. The only required parameter is the Prefactor :eq:`dipolar_prefactor`::
 
   from espressomd.magnetostatics import DipolarDirectSumGpu
   dds = DipolarDirectSumGpu(bjerrum_length=1)
   system.actors.add(dds)
 
 
-For testing purposes, a variant of the dipolar direct sum is available which adds periodic copies to the system in periodic directions (:class:`espressomd.magnetostatics.DipolarDirectSumWithReplicaCpu`).
+For testing purposes, a variant of the dipolar direct sum is available which adds periodic copies to the system in periodic directions (:class:`~espressomd.magnetostatics.DipolarDirectSumWithReplicaCpu`).
 
-As it is very slow, this method is not intended to do simulations, but
-rather to check the results you get from more efficient methods like
-P3M.
+The dipolar direct sum methods are very slow, and the CPU-based implementations
```

Comment:
The slowness comment was meant for the direct sum with replica spcifically. For open boundary systems, there is no choice but to use direct summation (except P2NFFT provided by the Scafacos library, which is more difficult to use).

Reply:
Thanks! It's fixed.